secretmanager/src/test/java/secretmanager/regionalsamples/SnippetsIT.java

```diff
@@ -0,0 +1,479 @@
+  private static SecretVersion TEST_REGIONAL_SECRET_VERSION_TO_DISABLE;
+  private static SecretVersion TEST_REGIONAL_SECRET_VERSION_TO_DISABLE_WITH_ETAG;
+  private static SecretVersion TEST_REGIONAL_SECRET_VERSION_TO_ENABLE;
+  private static SecretVersion TEST_REGIONAL_SECRET_VERSION_TO_ENABLE_WITH_ETAG;
+
+  private ByteArrayOutputStream stdOut;
+
+  @BeforeClass
+  public static void beforeAll() throws IOException {
+    Assert.assertFalse("missing GOOGLE_CLOUD_PROJECT", Strings.isNullOrEmpty(PROJECT_ID));
+    Assert.assertFalse("missing GOOGLE_CLOUD_PROJECT_LOCATION",
+        Strings.isNullOrEmpty(LOCATION_ID));
+
+    TEST_REGIONAL_SECRET = createRegionalSecret();
+    TEST_REGIONAL_SECRET_TO_DELETE = createRegionalSecret();
+    TEST_REGIONAL_SECRET_TO_DELETE_WITH_ETAG = createRegionalSecret();
+    TEST_REGIONAL_SECRET_WITH_VERSIONS = createRegionalSecret();
+    TEST_REGIONAL_SECRET_TO_CREATE_NAME = 
+        SecretName.ofProjectLocationSecretName(PROJECT_ID, LOCATION_ID, randomSecretId());
+
+    TEST_REGIONAL_SECRET_VERSION = addRegionalSecretVersion(TEST_REGIONAL_SECRET_WITH_VERSIONS);
+    TEST_REGIONAL_SECRET_VERSION_TO_DESTROY = 
+        addRegionalSecretVersion(TEST_REGIONAL_SECRET_WITH_VERSIONS);
+    TEST_REGIONAL_SECRET_VERSION_TO_DESTROY_WITH_ETAG = 
+        addRegionalSecretVersion(TEST_REGIONAL_SECRET_WITH_VERSIONS);
+    TEST_REGIONAL_SECRET_VERSION_TO_DISABLE = 
+        addRegionalSecretVersion(TEST_REGIONAL_SECRET_WITH_VERSIONS);
+    TEST_REGIONAL_SECRET_VERSION_TO_DISABLE_WITH_ETAG = 
+        addRegionalSecretVersion(TEST_REGIONAL_SECRET_WITH_VERSIONS);
+    TEST_REGIONAL_SECRET_VERSION_TO_ENABLE = 
+        addRegionalSecretVersion(TEST_REGIONAL_SECRET_WITH_VERSIONS);
+    TEST_REGIONAL_SECRET_VERSION_TO_ENABLE_WITH_ETAG = 
+        addRegionalSecretVersion(TEST_REGIONAL_SECRET_WITH_VERSIONS);
+    disableRegionalSecretVersion(TEST_REGIONAL_SECRET_VERSION_TO_ENABLE);
+    TEST_REGIONAL_SECRET_VERSION_TO_ENABLE_WITH_ETAG = disableRegionalSecretVersion(
+    TEST_REGIONAL_SECRET_VERSION_TO_ENABLE_WITH_ETAG);
+  }
+
+  @Before
+  public void beforeEach() {
+    stdOut = new ByteArrayOutputStream();
+    System.setOut(new PrintStream(stdOut));
+  }
+
+  @After
+  public void afterEach() {
+    stdOut = null;
+    System.setOut(null);
+  }
+
+  @AfterClass
+  public static void afterAll() throws IOException {
+    Assert.assertFalse("missing GOOGLE_CLOUD_PROJECT", Strings.isNullOrEmpty(PROJECT_ID));
+
+    deleteRegionalSecret(TEST_REGIONAL_SECRET.getName());
+    deleteRegionalSecret(TEST_REGIONAL_SECRET_TO_CREATE_NAME.toString());
+    deleteRegionalSecret(TEST_REGIONAL_SECRET_TO_DELETE.getName());
+    deleteRegionalSecret(TEST_REGIONAL_SECRET_TO_DELETE_WITH_ETAG.getName());
+    deleteRegionalSecret(TEST_REGIONAL_SECRET_WITH_VERSIONS.getName());
+  }
+
+  private static String randomSecretId() {
+    Random random = new Random();
+    return "java-" + random.nextLong();
```

Comment:
```suggestion
    return "test-drz-" + random.nextLong();
```

Reply:
Only Java samples will run in the project. We need a way to identify which test file a created resource belongs to.

Reply:
Fixed